web-backend/src/main/resources/application.yml

```diff
@@ -41,4 +41,14 @@ gg:
 # Application specific properties
 vms:
   system-email: test@email.com
-  admin-email: test@email.com
\ No newline at end of file
+  admin-email: test@email.com
+
+app:
+  email:
+    sender: noreply@company.com
+    visitor-confirmation:
+      subject: Registration Successful
+      template: visitorConfirmation.txt
```

Comment:
I'm not convinced about putting template location in here.
If you need to override it you'll still need to make code changes to add the new template and rebuild the application? what value does it add to put it in config?